sentry-spring-boot-starter/src/test/kotlin/io/sentry/spring/boot/SentryAutoConfigurationTest.kt

```diff
@@ -87,7 +87,7 @@ class SentryAutoConfigurationTest {
         contextRunner.withPropertyValues(
             "sentry.dsn=http://key@localhost/proj",
             "sentry.read-timeout-millis=10",
-            "sentry.shutdown-timeout-millis=20",
+            "sentry.shutdown-timeout=20",
```

Comment:
M: Should we name it millis instead for consistency?

Reply:
Yes but then we need to rename getter and setter too.

Reply:
@maciejwalkowiak I'd rename the getters to keep consistency, as all the android confs on the manifest ends with `-millis` as well.
we could keep the current getters/setters as aliases and annotate them with:

```
@ApiStatus.ScheduledForRemoval
@Deprecated
```

or maybe not even that, as this is a major bump.